go.mod

```diff
@@ -78,3 +78,9 @@ require (
 	sigs.k8s.io/randfill v1.0.0 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.6.0 // indirect
 )
+
+// TODO(mjudeikis): Remove this once kubernetes-sigs/multicluster-runtime cuts new release.
+// https://github.com/kcp-dev/multicluster-provider/issues/43 
+replace sigs.k8s.io/multicluster-runtime => github.com/kubernetes-sigs/multicluster-runtime v0.0.0-20251002124257-36facc7fbe82
```

Comment:
Can't we simply update this in the dependencies? Why do we need to do a `replace`?